### PR TITLE
Rollup DIQ to use shared package

### PR DESCRIPTION
**Pull request by Alixers505**
https://www.pivotaltracker.com/story/show/171677227
and
https://www.pivotaltracker.com/story/show/171698707

### CODE REVIEW

File: sites/dentistryiq.com/config/gcse.js [hunk 1]
```diff
@@ -1,5 +1,3 @@
-const GCSEConfiguration = require('@base-cms/marko-web-gcse/config');
+const configureGCSE = require('@endeavor-business-media/package-shared/config/gcse');
 
-const config = new GCSEConfiguration('003355913687346718228:prps-rbhnq0');
-
-module.exports = config;
+module.exports = configureGCSE({ apiKey: '003355913687346718228:266mmrxk-zy' });
```
Comment (zarathustra323) [inline sites/dentistryiq.com/config/gcse.js]: The `apiKey` is different. Is this intentional?

File: sites/dentistryiq.com/server/routes/content.js [hunk 2]
```diff
@@ -1,18 +1,13 @@
 const { withContent } = require('@base-cms/marko-web/middleware');
+const queryFragment = require('@endeavor-business-media/package-shared/graphql/fragments/content-page');
```
Comment (zarathustra323) [inline sites/dentistryiq.com/server/routes/content.js]: Use the same routing logic as DE (and dleete the local whitepaper template)

```marko
const { withContent } = require('@base-cms/marko-web/middleware');
const queryFragment = require('@endeavor-business-media/package-shared/graphql/fragments/content-page');
const companyQueryFragment = require('@endeavor-business-media/package-shared/graphql/fragments/content-company-page');
const noAdsLoadMore = require('@endeavor-business-media/package-shared/templates/content/no-ads-load-more');
const company = require('@endeavor-business-media/package-shared/templates/content/company');

const content = require('../templates/content');

module.exports = (app) => {
  app.get('/*?company/:id(\\d{8})*', withContent({
    template: company,
    queryFragment: companyQueryFragment,
  }));
  app.get('/*?webinar/:id(\\d{8})*', withContent({
    template: noAdsLoadMore,
    queryFragment,
  }));
  app.get('/*?whitepaper/:id(\\d{8})*', withContent({
    template: noAdsLoadMore,
    queryFragment,
  }));
  app.get('/*?:id(\\d{8})*', withContent({
    template: content,
    queryFragment,
  }));
};
```

File: sites/dentistryiq.com/config/gcse.js (hunk 1 above)
Comment (Alixers505) [reply sites/dentistryiq.com/config/gcse.js]: It was not. I'll get that corrected.

File: sites/dentistryiq.com/server/routes/content.js (hunk 2 above)
Comment (Alixers505) [reply sites/dentistryiq.com/server/routes/content.js]: pushing up now

**Comment by Alixers505** [comment]
Rebased branch to resolve conflicts

File: sites/dentistryiq.com/package.json [hunk 3]
```diff
@@ -17,23 +17,13 @@
...
-    "@base-cms/marko-web-social-sharing": "^1.25.0",
     "@base-cms/marko-web-theme-default": "^1.25.0",
     "@base-cms/object-path": "^1.25.0",
     "@base-cms/utils": "^1.25.0",
     "@base-cms/web-cli": "^1.25.0",
-    "@endeavor-business-media/package-common": "^1.10.0",
-    "@endeavor-business-media/package-inquiry": "^1.10.0",
     "@endeavor-business-media/package-leaders": "^1.10.0",
-    "@endeavor-business-media/package-photoswipe": "^1.10.0",
+    "@endeavor-business-media/package-shared": "^1.8.7",
```
Comment (zarathustra323) [inline sites/dentistryiq.com/package.json]: this should now be updated to `^1.10.0`

File: sites/dentistryiq.com/server/styles/index.scss [hunk 4]
```diff
@@ -22,17 +22,4 @@ $theme-site-header-breakpoints: (
   small-text-secondary: 598px
 );
```
Comment (zarathustra323) [inline sites/dentistryiq.com/server/styles/index.scss]: Remove `$theme-item-description-max-lines: 5;` from L15 (the core styles handle this now)

**Pull request merged**